src/Apps/Artwork/Components/ArtworkVideoPlayer/ArtworkVideoPlayer.tsx

```diff
@@ -35,10 +32,11 @@ const ArtworkVideoPlayer: FC<ArtworkVideoPlayerProps> = ({
       {...rest}
     >
       <ResponsiveBox
-        mx={[0, "70px"]}
-        maxWidth="100%"
-        aspectWidth={activeVideo.width}
-        aspectHeight={activeVideo.height}
+        maxWidth={MAX_DIMENSION}
+        maxHeight={MAX_DIMENSION}
```

Comment:
Here we just use the `ResponsiveBox` to scale the video container since we can't do it via GraphQL or with the `resize` function. This will fit it inside of an 800x800px box and then responsively scale it down if it's smaller.